### ISSUE TITLE
(3.0.0a5) TypeError: _evaluate() missing 1 required positional argument: 'recursive_guard'

### ISSUE THREAD

**Issue by ghost** [aiogram/aiogram#437]
## Context

* Operating System: Windows 10 Pro 2004
* Python Version: Python 3.9.0 (venv)
* aiogram version: 3.0.0a5
* aiohttp version: 3.6.2
* uvloop version (if installed): not installed

P.S. Everything works fine on stable aiogram version though.

## Expected Behavior

The application should start without exceptions.

## Current Behavior

An exception is thrown. 

### Steps to Reproduce

1. Write a basic bot code 
2. Try to run the code

<details>
  <summary>Example code</summary>

```python
import logging

from aiogram import Dispatcher, Bot

TOKEN = "2131243:asdasd"  # Obviously you need to change it

logging.basicConfig(level="DEBUG")

dp = Dispatcher()
bot = Bot(TOKEN)


@dp.message()
async def testing(msg):
    await msg.answer("hi")


dp.run_polling(bot)
```

</details>

### Failure Logs

```
Traceback (most recent call last):
  File "D:\Users\redacted\Desktop\wadsasd\app.py", line 3, in <module>
    from aiogram import Dispatcher, Bot
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\aiogram\__init__.py", line 1, in <module>
    from .api import methods, types
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\aiogram\api\methods\__init__.py", line 1, in <module>
    from .add_sticker_to_set import AddStickerToSet
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\aiogram\api\methods\add_sticker_to_set.py", line 5, in <module>
    from ..types import InputFile, MaskPosition
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\aiogram\api\types\__init__.py", line 218, in <module>
    _entity.update_forward_refs(**globals())
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\pydantic\main.py", line 677, in update_forward_refs
    update_field_forward_refs(f, globalns=globalns, localns=localns)
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\pydantic\typing.py", line 233, in update_field_forward_refs
    field.type_ = evaluate_forwardref(field.type_, globalns, localns or None)
  File "D:\Users\redacted\Desktop\wadsasd\venv\lib\site-packages\pydantic\typing.py", line 50, in evaluate_forwardref
    return type_._evaluate(globalns, localns)
TypeError: _evaluate() missing 1 required positional argument: 'recursive_guard'
```

Please include any relevant log snippets or files here.

**Comment by JrooTJunior** [comment]
Seems like pydantic was broken on Python 3.9, lol
I need to check it. It works as well on 3.7 and 3.8.

**Comment by uwinx** [comment]
Waiting for https://github.com/samuelcolvin/pydantic/pull/1844

**Comment by evgfilim1** [comment]
Now it works, but the fix requires updating to Pydantic 1.7+
Our lockfile still has 1.5.1

**Comment by evgfilim1** [comment]
Seems like #469 fixed this
